pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -1393,7 +1393,37 @@ def _check_unnecessary_comprehension(self, node):
         else:
             return
         if expr_list == target_list != []:
-            self.add_message("unnecessary-comprehension", node=node)
+            args = None
```

Comment:
```suggestion
            args: Optional[str] = None
```

Reply:
``Optional[Tuple[str]]`` instead, since args is = ``(msg,)``

Fixed